src/java/org/apache/cassandra/index/sai/disk/vector/CassandraOnHeapGraph.java

```diff
@@ -529,7 +530,7 @@ private long writePQ(SequentialWriter writer, V5VectorPostingsWriter.RemappedPos
 
         // Build encoder and compress vectors
         VectorCompressor<?> compressor; // will be null if we can't compress
-        Object encoded = null; // byte[][], or long[][]
+        CompressedVectors cv = null; // byte[][], or long[][]
```

Comment:
is the comment still useful "// byte[][], or long[][]" ?

Reply:
You're right, it's not useful anymore. I'll remove it and then merge once tests pass.